### PR TITLE
feat(docviewer): Toggle find bar control

### PR DESCRIPTION
**Pull request by ConradJChan**
Screen Shot 2019-10-24 at 4 22 01 PM

And don't worry about the consistency with the zoom in/out icons because they'll be changing

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @ConradJChan has signed the CLA. Thanks for the pull request!

File: src/lib/viewers/doc/DocBaseViewer.js [hunk 1]
```diff
@@ -771,16 +772,26 @@ class DocBaseViewer extends BaseViewer {
...
-        this.findBar = new DocFindBar(this.findBarEl, this.pdfFindController, this.pdfEventBus, canDownload);
+        this.findBar = new DocFindBar(this.findBarEl, this.pdfFindController, this.pdfEventBus);
         this.findBar.addListener(VIEWER_EVENT.metric, this.emitMetric);
         this.findBar.addListener('close', this.handleFindBarClose);
     }
 
+    /**
+     * Determines if findbar is disabled
+     *
+     * @protected
```
Comment (mxiao6) [inline src/lib/viewers/doc/DocBaseViewer.js]: Should this be `@private` since we only use this function within this class?

**Comment by mickr** [comment]
I think this may need to be rebased

File: src/lib/icons/search_16px.svg [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="18" height="18" viewBox="0 0 15 14">
+    <g fill="none" fill-rule="evenodd">
```
Comment (jstoffan) [inline src/lib/icons/search_16px.svg]: Is this group needed?

File: src/i18n/en-US.properties [hunk 3]
```diff
@@ -38,6 +38,8 @@ text_truncated=This file has been truncated due to size limits. Please download
 toggle_thumbnails=Toggle thumbnails
 # Message when file has inaccessible xrefs
 has_x_refs=This preview has references you cannot view. Open the file in its native application.
+# Button tooltip for toggling find bar
+toggle_findbar=Toggle findbar
```
Comment (jstoffan) [inline src/i18n/en-US.properties]: Minor, but can this live up next to `toggle_thumbnails`?

File: src/lib/viewers/doc/DocFindBar.js [hunk 4]
```diff
@@ -401,6 +401,19 @@ class DocFindBar extends EventEmitter {
...
         this.bar.classList.add(CLASS_HIDDEN);
     }
+
+    /**
+     * Toggles the Find Bar open and closed
+     *
+     * @return {void}
+     */
+    toggle() {
+        if (!this.opened) {
```
Comment (jstoffan) [inline src/lib/viewers/doc/DocFindBar.js]: We usually want to make the main condition positive, so `if (this.opened)` in this case.

File: src/lib/icons/search_16px.svg (hunk 2 above)
Comment (ConradJChan) [reply src/lib/icons/search_16px.svg]: Yeah, a black background appears behind the search icon without it

File: src/lib/viewers/doc/DocBaseViewer.js [hunk 5]
```diff
@@ -774,16 +775,26 @@ class DocBaseViewer extends BaseViewer {
...
     }
 
+    /**
+     * Determines if findbar is disabled
+     *
+     * @private
+     * @return {boolean}
+     */
+    isFindDisabled() {
+        const canDownload = checkPermission(this.options.file, PERMISSION_DOWNLOAD);
```
Comment (mickr) [inline src/lib/viewers/doc/DocBaseViewer.js]: very minor: I think the intention with this `const` is to be more readable so I would probably add the negation in the const creation and name it as such.

**Pull request merged**